Demo

Michael-Abebe opened the pull request.

Commit:
RWE first take

Commit:
Merge remote-tracking branch 'origin/master'

djarek commented:
Not sure if you're trying to upload something new, but it seems you don't have the proper history from master. Try doing a rebase.

Commit:
Corrected models

Corrected models according to newest database design
Added static
Chnged layout
